teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    @specialization = specialization
+    super
+  end
+
+  def can_use_services?
+    puts true
```

Comment:
- According to tp the project requirements, please return the value and not print it. Please fix this 🙏